raiden/blockchain/abi.py

```diff
@@ -134,6 +136,13 @@ def validate_solc():
             "Make sure the solidity compiler is installed and available on your $PATH."
         )
 
+    elif LooseVersion(_solidity.compiler_version()) < LooseVersion(MIN_REQUIRED_SOLC[1:]):
+        raise RuntimeError(
+            'You are currently using the solidity compiler version {0}.\n'
+            'Please upgrade to the compatible version {1}.'.format(
```

Comment:
Please upgrade to the compatible version  -> Please upgrade to the minimum compatible version